src/shared/elf-util.c

```diff
@@ -293,11 +293,13 @@ static char* build_package_reference(
 static void report_module_metadata(StackContext *c, const char *name, JsonVariant *metadata) {
         assert(c);
         assert(name);
-        assert(metadata);
 
         if (!c->f)
                 return;
 
+        if (!metadata)
+                return;
```

Comment:
is this the right approach? if we have neither buildid nor the packaging info, should we hide the module name or show it? with this change you hide it altogether?

@keszybz what is the intended behaviour here?

Reply:
I think we should still print the module name. It is useful, even if only to figure out what modules are without appropriate metadata.

Reply:
Now the module name is printed even if the JSON variant is NULL. PTAL.